Packs/Forcepoint_Security_Management_Center/Integrations/Forcepoint_Security_Management_Center/Forcepoint_Security_Management_Center.py

```diff
@@ -0,0 +1,240 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+import smc
+from smc import session
+from smc.elements.network import IPList, DomainName
+from smc.policy.layer3 import FirewallTemplatePolicy, FirewallPolicy
+from smc.policy.rule import Rule
+from smc.api.exceptions import ElementNotFound
+import urllib3
+from typing import Dict, Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+DEFAULT_LIMIT = 50
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
```

Comment:
any reason to inherit?
```suggestion
class Client:
```